torch/_export/wrappers.py

```diff
@@ -115,3 +126,96 @@ def call(self, *args):
 
     cls.__call__ = call
     return cls
+
+
+def _register_and_get_spec_proxy_in_tracer(tracer, name, spec):
```

Comment:
can you explain what this is for

Reply:
Not needed! The context is i wanted to reuse the spec between different Dtensor init but i realized DeviceMesh etc can be different so it doesn't make sense to use same spec

Reply:
Actually we do need it for registering subclass constructor in fx graph and not duplicate it :)

Reply:
hmm I guess it's not at all clear from the name / description of this fn that its main purpose is to take subclass constructor functions and put them as constant attributes on the graph. I guess it's theoretically more general than that, but it might be clearer to specialize the name etc just to make it clearer what the purpose of this function is?

Also - I think the expectation here is that nobody should be serializing/deserializing graphs with subclass constructors in it (at least, until we get the autograd HOP for subclasses working). Do you agree? And if so, can we add a loud error somewhere and a test for it, that way users get a good error message?

Reply:
Yep updated the name! Today, when people try to export a graph with subclass constructor and ser/der, it will error complaining it is not supported. This is because it will try to serialize Spec object directly. (see test_export.py expectedFailure)